cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -1459,6 +1459,17 @@ def _add_compute_resource_launch_template(
         if isinstance(compute_resource, SlurmComputeResource):
             conditional_template_properties.update({"instance_type": compute_resource.instance_type})
 
+        capacity_reservation_specification = None
+        if isinstance(queue, (SlurmQueue, SchedulerPluginQueue)):
```

Comment:
Is this check `f isinstance(queue, (SlurmQueue, SchedulerPluginQueue))` necessary? 
`def _add_compute_resource_launch_template` function is under class `ComputeFleetConstruct`, ComputeFleetConstruct is not for awsbatch scheduler.
See code:
```
        if not self._condition_is_batch():
            self.compute_fleet_resources = ComputeFleetConstruct(
```

Reply:
Done